crates/wascap/src/caps.rs

```diff
@@ -0,0 +1,33 @@
+//! A set of standard names for capabilities that can be provided by a host
+
+use std::collections::HashMap;
+
+use once_cell::sync::Lazy;
+
+pub const BLOB: &str = "wasmcloud:blobstore";
+pub const HTTP_CLIENT: &str = "wasmcloud:httpclient";
+pub const HTTP_SERVER: &str = "wasmcloud:httpserver";
+pub const KEY_VALUE: &str = "wasmcloud:keyvalue";
+pub const MESSAGING: &str = "wasmcloud:messaging";
+pub const EVENTSTREAMS: &str = "wasmcloud:eventstreams";
+pub const NUMBERGEN: &str = "wasmcloud:builtin:numbergen";
+pub const LOGGING: &str = "wasmcloud:builtin:logging";
+
+static CAPABILITY_NAMES: Lazy<HashMap<&str, &str>> = Lazy::new(|| {
+    let mut m = HashMap::new();
+    m.insert(MESSAGING, "Messaging");
+    m.insert(KEY_VALUE, "K/V Store");
+    m.insert(HTTP_SERVER, "HTTP Server");
+    m.insert(HTTP_CLIENT, "HTTP Client");
+    m.insert(BLOB, "Blob Store");
+    m.insert(EVENTSTREAMS, "Event Streams");
+    m.insert(NUMBERGEN, "Number Generation");
+    m.insert(LOGGING, "Logging");
+    m
+});
+
+pub fn capability_name(cap: &str) -> String {
+    CAPABILITY_NAMES
+        .get(cap)
+        .map_or(cap.to_string(), |item| item.to_string())
+}
```

Comment:
```suggestion
//! A set of standard names for capabilities that can be provided by a host

use std::sync::OnceLock;

pub const BLOB: &str = "wasmcloud:blobstore";
pub const HTTP_CLIENT: &str = "wasmcloud:httpclient";
pub const HTTP_SERVER: &str = "wasmcloud:httpserver";
pub const KEY_VALUE: &str = "wasmcloud:keyvalue";
pub const MESSAGING: &str = "wasmcloud:messaging";
pub const EVENTSTREAMS: &str = "wasmcloud:eventstreams";
pub const NUMBERGEN: &str = "wasmcloud:builtin:numbergen";
pub const LOGGING: &str = "wasmcloud:builtin:logging";

use std::collections::HashMap;

static CAPABILITY_NAMES: OnceLock<HashMap<&str, &str>> = OnceLock::new();
fn get_capability_names() -> &'static HashMap<&'static str, &'static str> {
    CAPABILITY_NAMES.get_or_init(|| {
        HashMap::from([
            (MESSAGING, "Messaging"),
            (KEY_VALUE, "K/V Store"),
            (HTTP_SERVER, "HTTP Server"),
            (HTTP_CLIENT, "HTTP Client"),
            (BLOB, "Blob Store"),
            (EVENTSTREAMS, "Event Streams"),
            (NUMBERGEN, "Number Generation"),
            (LOGGING, "Logging"),
        ])
    })
}

pub fn capability_name(cap: &str) -> String {
    get_capability_names()
        .get(cap)
        .map_or(cap.to_string(), |item| item.to_string())
}
```

We can get rid of the `once_cell` dep here by using `std::sync::OnceLock`:

Reply:
That a great improvement, let's address in a follow-up and focus on getting this merged ASAP to prevent merge conflicts